sources/Cargo.toml

```diff
@@ -24,6 +24,8 @@ members = [
 
     "models",
 
+    "parse-datetime",
+
```

Comment:
Need to update `packages/os/Cargo.toml` also to watch this directory for changes.